sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/AzureCosmosDBChatExtensionParameters.java

```diff
@@ -31,14 +31,14 @@ public final class AzureCosmosDBChatExtensionParameters {
      * The configured top number of documents to feature for the configured query.
      */
     @Generated
-    @JsonProperty(value = "topNDocuments")
```

Comment:
Did the property names on the wire change or was this previous incorrect?

Reply:
previous incorrect